backend/app/views/kaminari/_paginator.html.erb

```diff
@@ -6,16 +6,18 @@
     remote:        data-remote
     paginator:     the paginator that renders the pagination tags inside
 -%>
-<%= paginator.render do %>
-  <ul class="pagination pagination-sm">
-    <%= prev_page_tag %>
-    <% each_page do |page| %>
-      <% if page.left_outer? || page.right_outer? || page.inside_window? %>
-        <%= page_tag page %>
-      <% elsif !page.was_truncated? %>
-        <%= gap_tag %>
-      <% end %>
-    <% end %>
-    <%= next_page_tag %>
-  </ul>
-<% end %>
+<%= paginator.render do -%>
+    <nav class="pagination pagination-sm">
+      <%= first_page_tag unless current_page.first? %>
+      <%= prev_page_tag unless current_page.first? %>
+      <% each_page do |page| -%>
+          <% if page.left_outer? || page.right_outer? || page.inside_window? -%>
```

Comment:
also it would be great if wrap condition in one helper method.
@damianlegawiec  what you say?